README.adoc

```diff
@@ -16,11 +16,20 @@ The following guides are available:
 * SUSE OpenStack Cloud, General:
 ** Security Guide
 ** Supplement to Administrator Guide and User Guide
+
+These guides are all maintained and created in this GitHub
+repository .
+
+The folowing guides are available upstream:
+
 ** Administration Guide
 ** User Guide
 
-Released versions of these guides have been published at
-https://www.suse.com/documentation/.
+These guides are all maintained upstream in the various OpenStack
+project repositories. Released versions of these guides have been
+generated using the `suse_sphinx_theme` and published at
+https://www.suse.com/documentation/. This setup can be found
+here: https://github.com/SUSE-Cloud/doc-cloud-upstream
```

Comment:
This is a bit unclear, maybe make two paragraphs out of it, as:

* only the upstream stuff is built with the SUSE Sphinx theme
* both SUSE and upstream guides are published at the same location